include/seastar/http/request.hh

```diff
@@ -60,6 +60,7 @@ struct request {
     sstring _version;
     ctclass content_type_class;
     size_t content_length = 0;
+    mutable size_t _real_content_length = 0;
```

Comment:
i'd suggest change the variable name to `_bytes_written`. because it is not the length of the request's content. instead, it is used to tell the write ptr's position. yeah, it is also used for passing the length to the data sink. so it's both an input parameter **and** an output parameter.

this is both clever and confusing. this is more related to the design of `http_content_length_data_sink_impl` , and is not in the scope of this change. but i think we can make it less confusing with a better name.

Reply:
i created #1849 to explain what i am thinking.

Reply:
OK, renamed